testing/librsynctest.py

```diff
@@ -11,12 +11,10 @@ def MakeRandomFile(path, length=None):
     if not length:
         length = random.randrange(5000, 100000)
     fp = open(path, "wb")
```

Comment:
This is already a lot cleaner, to make it even cleaner, you could write something like:
```
with open(...) as fp:
    fp.write(...)
```
then you don't even need to explicitly close the file handler `fp`.

Reply:
Reference: https://docs.python.org/3/tutorial/inputoutput.html#reading-and-writing-files